op-conductor/health/monitor.go

```diff
@@ -167,6 +171,7 @@ func (hm *SequencerHealthMonitor) healthCheck() error {
 			"unsafe_head_num", status.UnsafeL2.Number,
 			"unsafe_head_time", status.UnsafeL2.Time,
 			"unsafe_interval", hm.unsafeInterval,
+			"curr_unsafe_timeDiff", calculateTimeDiff(now, status.UnsafeL2.Time),
```

Comment:
`calculateTimeDiff(now, status.UnsafeL2.Time)` used multiple times here, put it into a variable